schedule/migration/openldap_to_389ds.yaml

```diff
@@ -0,0 +1,7 @@
+name:   openldap_to_389ds
+description:    >
+    This is for openldap to 389ds migration tools test
```

Comment:
That should be rewritten not to say "This is" but rather what this scenario does and what it covers